kafka/pykafka/broker.py

```diff
@@ -77,7 +77,7 @@ def connect(self):
     def fetch_messages(self,
                        partition_requests,
                        timeout=30000,
-                       min_bytes=1024):
+                       min_bytes=1):
```

Comment:
I'm not sure if this page is outdated, but I found 1 here https://kafka.apache.org/08/configuration.html (search for fetch.min.bytes)